src/ngraph/op/interpolate.cpp

```diff
@@ -32,8 +31,21 @@ op::Interpolate::Interpolate(const Output<Node>& image,
     constructor_validate_and_infer_types();
 }
 
+bool op::Interpolate::visit_attributes(AttributeVisitor& visitor)
```

Comment:
Please change `op::Interpolate*` to `op::v0::Interpolate*` in this file so it's easy to tell what version everything in.